### PR TITLE
City Entity Attributes and Offsets

### PR DESCRIPTION
**Pull request by tthordarson**
Note that offsets are only tested on Steam 15_3_2 Windows. If they differ for Linux or Mac then we need to replace the constants inside CityOffsets with properties that use the same names with conditions as are done in some of the other offset classes.

### CODE REVIEW

**Comment by tthordarson** [comment]
Can we get the unit test to build in Travis or should I remove it altogether? I personally find it preferable to keep it there.

**Comment by ThanosSiopoudis** [comment]
Sorry, but Microsoft's Unit Test Framework is not available on any platform other than windows. Consider using [NUnit](http://www.mono-project.com/archived/nunit/) which is available on all platforms, or remove it all together. Thanks

File: FMScoutFramework.sln [hunk 1]
```diff
@@ -1,12 +1,18 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
+<<<<<<< HEAD
```
Comment (ThanosSiopoudis) [inline FMScoutFramework.sln]: It looks like you have commited an unresolved conflict. This will break the solution file. Please resolve before merging.

**Comment by tthordarson** [comment]
I tried to reverse the solution file back to how it is on your repository and I removed the unit test project. Are you able to merge now?

**Pull request closed without merging**